<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="portChoseWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QPushButton" name="portPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose port</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bitrateBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Bitrate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="portBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="portRefreshButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="plotDataCleaner">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Data</string>
     </property>
    </widget>
   </widget>
   <widget class="MyPlot" name="thePlot" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>681</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="plotScrollBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="oxScaleBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>90000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OX scale:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="oxAccessibleArea">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="singleStep">
     <number>1000</number>
    </property>
    <property name="value">
     <number>20000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OX range:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPlot</class>
   <extends>QWidget</extends>
   <header>plot/myplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>portBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>portAvailability()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrateBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>portAvailability()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>portPushButton</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>portChosen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>portRefreshButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshPorts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plotDataCleaner</sender>
   <signal>clicked()</signal>
   <receiver>thePlot</receiver>
   <slot>clearData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>portAvailability()</slot>
  <slot>portChosen()</slot>
  <slot>refreshPorts()</slot>
 </slots>
</ui>
